react/react1/week2/todo-list/src/components/TodoList.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { TodoItem } from "./TodoItem";
+import todos from "./todos";
+import "../App.css";
+
+export function TodoList() {
+  const [useTodo, setUseTodo] = useState(todos);
+
+  const addRandomTodo = () => {
+    const myNextTodoItem = "random things";
+    const newTodo = {
+      id: useTodo.length + 1,
```

Comment:
There is a small bug here in certain cases by using the list's length to create a new id.
Let's say you have todos with 1,2 and 3 ids.
Then you add a new one which gets the id 4.
Then you decide to remove the todo which has id 3.
If you try to add new todo now, it will get the id 4 again since the length of the list is 3

So, you could either generate a random id, or you could use the last todo's id in the list and increment it

Reply:
I didn't notice, now it's done :)